src/components/Form.js

```diff
@@ -0,0 +1,51 @@
+import { useState } from "react"
+import '../styles/_Form.scss'
+const validator = require("email-validator");
+
+const Form = () => {
+  const [email, setEmail] = useState('')
+  const [firstName, setFirstName] = useState('')
+  const [lastName, setLastName] = useState('')
+  const [comments, setComments] = useState('')
+  const [emailError, setEmailError] = useState(false)
+  const [firstNameError, setFirstNameError] = useState(false)
+  const [lastNameError, setLastNameError] = useState(false)
+
+  const submitRequest = (e) => {
+    e.preventDefault()
+    if (!validator.validate(email)) {
+      setEmailError(true)
+    } else {
+      setEmailError(false)
+    }
+    if (!firstName) {
+      setFirstNameError(true)
+    } else {
+      setFirstNameError(false)
+    }
+    if (!lastName) {
+      setLastNameError(true)
+    } else {
+      setLastNameError(false)
+    }
+  }
```

Comment:
I might not be able to phrase this properly but I anticipate a refactor for this when it comes to actually sending a mutation request. I think you'd need an overall "formHealth" state (which could basically be all the booleans separated by && (ie. emailError && firstNameError && lastNameError). Depending on THAT being true/false, you might then go ahead with the mutation. I loosely anticipate this so it's not a request to change anything but I know you love to think critically about future code needs. Would love to know your thoughts.

Reply:
yeah, I agree, something like that will definitely need to be implemented when I actually do the mutation